articles/synapse-analytics/overview-faq.yml

```diff
@@ -75,14 +75,19 @@ sections:
           Azure Synapse Link for SQL Server 2022 is in preview. For more information, see [What is Azure Synapse Link for SQL?](synapse-link/sql-synapse-link-overview.md).
 
       - question: |
-          Is Azure Synapse Link to Azure SQL Database generally available?
+          Does Azure Synapse workspace Support CI/CD? 
         answer: |
-          Azure Synapse Link for Azure SQL Database is in preview. For more information, see [What is Azure Synapse Link for SQL?](synapse-link/sql-synapse-link-overview.md).
+          Yes! All Pipeline artifacts, notebooks, SQL scripts, and Spark job definitions will reside in Git. All pool definitions will be stored in Git as Azure Resource Manager (ARM) templates. Dedicated SQL pool objects (schemas, tables, views, etc.) will be managed with database projects with CI/CD support.
```

Comment:
Link to docs https://docs.microsoft.com/en-us/azure/synapse-analytics/cicd/continuous-integration-delivery

Reply:
Added in commit cb70304